src/python/tensorflow_cloud/core/deploy.py

```diff
@@ -12,20 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Utilities for deploying jobs to GCP."""
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
 
 import subprocess
 import uuid
 
 from googleapiclient import discovery
 from googleapiclient import errors
-from googleapiclient import http as googleapiclient_http
 
 from . import gcp
 from . import machine_config
-from .. import version
+from ..utils import google_api_client
```

Comment:
Do you strongly prefer relative import, compared to absolute import?

from tensorflow_cloud.utils import google_api_client

Reply:
No, i did not have a preference. I will refactor everything to absolute imports since this works better with copybara.

Reply:
Python-wise, we should prefer relative in this case. Is this a problem with Copybara?

Reply:
Adding this to meeting agenda and resolving this thread here.

Reply:
It is rather Blaze/Bazel that relative import is not very friendly to relative import. I believe for this reason, TensorFlow repositories that use Bazel follows absolute import style.

Copybara could rewrite relative import to absolute, but it would be like writing import statement twice.